### PR TITLE
Revamp the Translation Program page

### PR DESCRIPTION
**Pull request by lukassim**
## Description

- Add acknowledgements section to the Translation Program page

## Related Issue

Part of the _Revamp the Translation Program page_ epic
#7206

### CODE REVIEW

File: src/content/contributing/translation-program/index.md [hunk 1]
```diff
@@ -36,19 +36,30 @@ Read more about the ethereum.org Translation Program [mission and vision](/contr
...
-If you want to get involved and help us grow the global Ethereum community by translating the website into your language, follow the steps below!
+- [**2.2 million** words translated in 2022 so far](/contributing/translation-program/acknowledgements/)
 
 <InfoBanner emoji=":tada:">
   Check out our <Link to="/contributing/translation-program/acknowledgements/">Translator Acknowledgements</Link> page, and{" "}
   claim your POAP token! If you translated ethereum.org in 2022, there's a unique POAP waiting for you.{" "}
   <Link to="/contributing/translation-program/acknowledgements/#poap">More on POAPs</Link>
 </InfoBanner>
 
+#### Acknowledgements {#acknowledgements}
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:49]: This should be an H3

```suggestion
### Acknowledgements {#acknowledgements}
```

File: src/content/contributing/translation-program/index.md [hunk 2]
```diff
@@ -36,19 +36,30 @@ Read more about the ethereum.org Translation Program [mission and vision](/contr
...
+
+Ethereum.org is translated by thousands of community members and they are the key part of the Translation Program.
+We want to acknowledge our translators and support them on their career paths. Here are some of our translator acknowledgements:
+
+**Certificate**
```
Comment (minimalsm) [inline src/content/contributing/translation-program/index.md:54]: This should be an H4

**Comment by minimalsm** [comment]
We have uploaded the assets twice in this PR, we should remove one of the two instances.

File: src/components/TranslationChartImage.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+export interface IProps {}
+
+const TranslationChartImage: React.FC<IProps> = () => {
+  const { colorMode } = useColorMode()
+  const isDarkTheme = colorMode === "dark"
+
+  const data = useStaticQuery(graphql`
+    {
+      pageviewsLight: file(
+        relativePath: { eq: "translation-program/pageviews-light.png" }
```
[the remark this answers is not in the document]
Comment (minimalsm) [reply src/components/TranslationChartImage.tsx:17]: Yup you're correct 🙏

[the remark this answers is not in the document]
Comment (wackerow) [reply src/components/TranslationChartImage.tsx:17]: Just added a commit to remove these.

**Review by wackerow — APPROVED** [review]

**Pull request merged**